lib/run_cucumber.js

```diff
@@ -79,46 +79,52 @@ function build_config( command_args ) {
   let one_dir_exists = session_vars.set_sources_paths( sources_list );
   if ( !one_dir_exists ) {
     // If we don't throw, it'll be more confusing when no tests run
-    throw new ReferenceError( `ALKiln ERROR: ALKiln can't find your "sources" folder(s). See above for details.`);
+    log.error({ type: `run`, code: `ALK0049`, pre: `ALKiln can't find your "sources" folder(s). See above for details.` });
+    throw new ReferenceError(`ALK0050 ERROR: ALKiln can't find your "sources" folder(s). See above for details.`);
```

Comment:
These were supposed to be the same error code. Gorramit. We _could_ re-use one of those error codes at some future point. That's going to be hard to keep track of. We could just leave it in the dust because the other option is to re-number _all_ the error codes that come after. I may just be too compulsive to avoid it...

Reply:
Fixed